### PR TITLE
Removed reference to nonexistent network client

### PR DESCRIPTION
**Pull request by mchf**

### CODE REVIEW

**Comment by mvidner** [comment]
A changelog entry is missing. And don't just say "Removed reference to nonexistent network client". It only becomes nonexistent because of the other PR. Instead, repeat the **reason** why we are removing it.

**Comment by mchf** [comment]
that happens when I forget to do ```git push```. @mvidner: could you pls check the changelog?

File: package/skelcd-control-SLES.changes [hunk 1]
```diff
@@ -1,4 +1,12 @@
 -------------------------------------------------------------------
+Tue Oct 11 11:24:17 UTC 2016 - mfilka@suse.com
+
+- Removed reference to netprobe client. It was used in AutoYaST
+  only and everything what this client does should be currently
+  handled by wicked.
+- 12.2.3
```
Comment (mvidner) [inline package/skelcd-control-SLES.changes]: Hmm, this is not going to SP2, is it? Then we should use 12.3.0 to mark this as SP3.

Comment (mchf) [inline package/skelcd-control-SLES.changes]: ok

**Pull request merged**